tests/utils/test_uri.py

```diff
@@ -217,10 +224,7 @@ def test_append_to_uri_path_handles_special_uri_characters_in_posixpaths():
         "'",
         ",",
     ]:
-
-        def char_case(*case_args):
-            return tuple([item.format(c=special_char) for item in case_args])
```

Comment:
This violates: https://pycodequ.al/docs/pylint-messages/w0640-cell-var-from-loop.html